src/Params/WaveTable.h

```diff
@@ -269,12 +270,14 @@ class WaveTable
     Tensor3<float32> data;  //!< time=col,freq=row,semantics(oscil param or random seed)=depth
     WtMode m_mode;
 
+    std::atomic<std::size_t> reader;
```

Comment:
I'd avoid dealing with a size_t object here. the number of elements is almost certainly a value that can be stored in a short (or more simply a 32bit value). size_t is typically (from what I'm recalling) 64 bit which can lead to some headaches in 32bit or arm situations where atomic writes get messy for 64bit values.

Reply:
I'll leave this open as a reminder for later, too.

Reply:
@fundamental Is this comment concerned with atomics? In this case, we could stay with `std::size_t`, since the code only uses `uToB`/`bToU` for communication. Or are you more worried about performance?

Reply:
If uToB and bToU are the only ways to communicate then std::atomic isn't needed. If it is accessed across threads then it might be best to use a 32bit value.